tests/stdlib/tonce.nim

```diff
@@ -0,0 +1,34 @@
+discard """
+  cmd: "nim c -r --threads:on $options $file"
```

Comment:
add `# xxx: support vm+js`
(right now doesn't work, not even considering webworkers)